### DISCUSSION TITLE
Add 2nd color for numerical entries

### DISCUSSION THREAD

**Discussion by Thorn00** [iSoron/uhabits#1637] [Feature Requests]
I understand the app doesn't handle < targets.  For instance, a numerical entry such as weight.  Enter weight above target and entry is colored.  Enter weight below target and it's grayed out.  If 2nd color option is added,  then it could be primary habit color above target and a different color below target.  Thought this idea might be a quicker add than having it do <targets correctly.

**Comment by herrbrixter** [comment]
Related to #2304
